package.json

```diff
@@ -53,6 +53,8 @@
     "@types/react-select": "^1.0.59",
     "@types/uuid": "^2.0.30",
     "@types/webpack-env": "^1.13.1",
+    "@ycleptkellan/substantive": "^1.0.2",
```

Comment:
What's going on here? Is this a fork?

Reply:
i moved some helpers to their own library https://github.com/ycleptkellan/substantive